bpf/process/generic_calls.h

```diff
@@ -625,8 +625,12 @@ FUNC_INLINE int generic_process_filter(void)
 		return 0;
 
 	enter = event_find_curr(&ppid, &walker);
-	if (!enter)
-		return PFILTER_CURR_NOT_FOUND;
+	if (!enter) {
+		enter = event_find_curr_probe(msg);
+		if (!enter)
+			return PFILTER_CURR_NOT_FOUND;
```

Comment:
is it okay that this function always return `PFILTER_CURR_NOT_FOUND` on !__LARGE_BPF_PROG kernels? I guess this related to my other comment, there's something I don't understand I think